packages/@tailwindcss-vite/src/index.ts

```diff
@@ -113,16 +125,21 @@ export default function tailwindcss(): Plugin[] {
 
       async configResolved(config) {
         minify = config.build.cssMinify !== false
-        // Apply the vite:css plugin to generated CSS for transformations like
-        // URL path rewriting and image inlining.
-        //
-        // In build mode, since renderChunk runs after all transformations, we
-        // need to also apply vite:css-post.
-        cssPlugins = config.plugins.filter((plugin) =>
-          ['vite:css', ...(config.command === 'build' ? ['vite:css-post'] : [])].includes(
-            plugin.name,
-          ),
-        )
+        isSSR = config.build.ssr !== false && config.build.ssr !== undefined
```

Comment:
I was confused why the second undefined check was there, but didn't realise `ssr` could be a string.

Reply:
heh yeah I looked at the types and I was like… oh okay